backend/src/databasefunctions.ts

```diff
@@ -0,0 +1,80 @@
+import { PoolConfig, ClientConfig } from 'pg';
+
+const { Pool } = require('pg');
+require('dotenv').config();
+
+const pgconfig: PoolConfig = {
+  user: process.env.DBUSER,
+  database: process.env.DATABASENAME,
+  password: process.env.DBPASSWORD,
+  host: process.env.DBHOST,
+  port: parseInt(process.env.DBPORT),
+};
+
+const pool = new Pool(pgconfig);
+
+pool.on('error', (err: Error, client: ClientConfig) => {
+  console.error(err, client);
+});
+export const createTable = async (sqlStatement: string) => {
+  if (
```

Comment:
To me it looks like there is some redundant code in lines 20-25, 44-46 and 65-67.
The only difference i can see is that in the first lines you also check if sqlStatement is set (which you might as well do in the two other?). It might be an idea to abstract that code into it's own method. something like
```
const isValidSqlStatement = (sqlStatement: string) => {
// Check if valid
// If not, then throw error
}
```

I don't remember exactly how Express does it, but I'm assuming there is a way to pass the error all the way up the callstack to a centralized error handler. So that errors can be handled at one location and less and there is less error handling scattered across the code